src/Avalonia.Controls/PositionChangedEventArgs.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) The Avalonia Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+using System;
+
+namespace Avalonia.Controls
+{
+    /// <summary>
+    /// Provides data for the <see cref="TopLevel.PositionChanged"/> event.
+    /// </summary>
+    public class PositionChangedEventArgs : EventArgs
```

Comment:
This might be best called `PointEventArgs` - that way we can use it for any events that want to send a `Point`.